src/payment/strategies/braintree/braintree-paypal-payment-strategy.ts

```diff
@@ -123,12 +124,18 @@ export default class BraintreePaypalPaymentStrategy implements PaymentStrategy {
             throw new InvalidArgumentError('Vaulting is disabled but shouldSaveInstrument is set to true');
         }
 
+        const shippingAddress = state.shippingAddress.getShippingAddress();
+
+        const braintreeAddress: BraintreeAddress | undefined =
```

Comment:
Since the function is already typed, I think you can safely omit the types for this.
```suggestion
        const braintreeAddress = shippingAddress ? mapToBraintreeAddress(shippingAddress) : undefined;
```